ocs_samples/basic_samples/SDS/JavaScript/NodeJs/readme.md

```diff
@@ -329,6 +331,29 @@ restCall({
    format will be
    "/{streamId}/Data?startIndex={startTime}&endIndex={endTime}
 
+Getting sampled values:
+
+```js
+restCall({
+    url: this.url + this.streamsBase.format([tenantId, namespaceId]) + this.getSamplesBase.format([streamId, start, end, intervals, sampleBy, filter, streamViewId]),
```

Comment:
we include the optional filter parameter but not the optional boundary parameters. is this intentional?

Reply:
It is mentioned in the readme that the BoundaryType has been left out, I could take out the filter and streamViewId as well, the boundary types are 3 additional optional parameters and I had trouble with the requests when they were at first included but haven't gone back to look at them since.

Reply:
No that's fine, not a big deal.